installer.sh

```diff
@@ -145,7 +145,7 @@ if grep -q -i wsl /proc/version; then
 fi
 
 # distribution check
-if ! grep -q "ID_LIKE=debian" /etc/os-release 2>/dev/null ; then
+if ! grep -q "ID_LIKE=" /etc/os-release | grep -q "ubuntu\|debian" /etc/os-release 2>/dev/null ; then
```

Comment:
is there a copy error? Should this be something like (untested):
```
if ! grep -q "ID_LIKE=debian\|ID_LIKE=ubuntu" /etc/os-release
```

Could you provide a /etc/os-release file?

Reply:
Here is the content of /etc/os-release on my system:
NAME="Pop!_OS"
VERSION="22.04 LTS"
ID=pop
ID_LIKE="ubuntu debian"
PRETTY_NAME="Pop!_OS 22.04 LTS"
VERSION_ID="22.04"
HOME_URL="https://pop.system76.com"
SUPPORT_URL="https://support.system76.com"
BUG_REPORT_URL="https://github.com/pop-os/pop/issues"
PRIVACY_POLICY_URL="https://system76.com/privacy"
VERSION_CODENAME=jammy
UBUNTU_CODENAME=jammy
LOGO=distributor-logo-pop-os

Reply:
would this regex fit?
```
grep -E "ID_LIKE=(\")?(ubuntu)?( )?(debian)?"
```
Looks not too bad in my testset:

![image](https://user-images.githubusercontent.com/497520/215258067-f92c3f81-bfdd-46a1-b1b8-eb3f7f4033c7.png)

Reply:
Yes, it looks good. I was too lazy to write a regex I guess